backend/config.py

```
@@ -0,0 +1,21 @@
+
+# SECURITY VULNERABILITY - Hardcoded Qdrant credentials
+QDRANT_URL = "sample_url"
+QDRANT_API_KEY = "sample_key"
+
+# Database configuration
+DATABASE_PATH = "data/diy_finder.db"
+QDRANT_COLLECTION_NAME = "items"
+VECTOR_SIZE = 1024
+
+# Server configuration
+HOST = "0.0.0.0"
```

Ruff (0.16.0)

[error] Possible binding to all interfaces

(S104)